test/test_pre_symptom_prep.py

```diff
@@ -35,12 +35,12 @@ def test_drop_from_list(self, prep):
 
     def test_verify_answers_for_row(self, prep):
         headers = ['sid', 'test1', 'test2', 'test3']
-        row = dict(zip(headers, ['0', 1, 2, 2015]))
+        row = dict(list(zip(headers, ['0', 1, 2, 2015])))
```

Comment:
I really don't think this list() is necessary here and throughout this file where you have this pattern. I tested a bit on my own and got the same output with and without the list. The one thing that the linter I use suggests is setting `strict=True` inside the zip.